### ISSUE TITLE
Set different learning rate for different layer

### ISSUE THREAD

**Issue by HB-Liu** [open-mmlab/mmclassification#46]
How to set different learning rates for different layers?
Such as：
```
torch.optim.SGD([{'params':model.backbone.parameters(), 'lr': learning_rate*0.1},
                 {'params':model.clshead.parameters() ,'lr': learning_rate},
                 momentum=0.9, weight_decay=1e-4}])
```

**Comment by mzr1996** [comment]
You can specify detail learning policy setting for each layer and even each parameter by using `paramwise_cfg`,
For example,
```python
optimizer = dict(
    type='SGD',
    lr = 0.001,
    paramwise_cfg=dict(
        # The learning rate of `backbone.classifier` layer params will be multiplied 10.
        custom_keys={'.backbone.classifier': dict(lr_mult=10)}   
    )
)
```
More detail document refers to [mmcv docs](https://mmcv.readthedocs.io/en/latest/api.html?highlight=DefaultOptimizerConstructor#mmcv.runner.DefaultOptimizerConstructor)

**Comment by mzr1996** [comment]
Should have been solved.
